ocean/helium/snapshots.py

```diff
@@ -138,3 +138,69 @@ def download_bytes_from_snapshot(src, snapshothash):
     snapshot_data = read_snapshot_by_hash(bucket, snapshothash)
     obj_rec = snapshot_data['contents'][key]
     return download_bytes(src, version=obj_rec['VersionId'])
+
+
+class Snapshot(object):
+    """ In-memory representation of a snapshot """
+
+    def __init__(self):
+        """
+        _data is of the form {logical_key: entry}
+        entry is of the form (physical_key, hash, size, user_meta)
+        physical_key is of the form {
+            schema_version: string
+            type: string
+            uri: string
+        }
+        hash is of the form {
+            type: string (e.g. "SHA256")
+            value: string
+        }
+        size is the length of the object in bytes
+        """
+        self._data = {}
+        pass
+
+    @staticmethod
+    def load(path):
+        """ loads a snapshot from a path """
```

Comment:
Let's make all the docstrings sentences, capital letters to start and punctuation to end.